flink-tests/src/test/java/org/apache/flink/test/scheduling/AdaptiveSchedulerITCase.java

```diff
@@ -36,31 +41,48 @@
 import org.apache.flink.streaming.api.functions.source.RichParallelSourceFunction;
 import org.apache.flink.test.util.MiniClusterWithClientResource;
 import org.apache.flink.util.FlinkException;
+import org.apache.flink.util.Preconditions;
 import org.apache.flink.util.TestLogger;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 import javax.annotation.Nullable;
 
+import java.io.File;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
 import static org.junit.Assume.assumeTrue;
 
 /** Integration tests for the adaptive scheduler. */
 public class AdaptiveSchedulerITCase extends TestLogger {
 
+    @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
     private static final int NUMBER_TASK_MANAGERS = 2;
     private static final int NUMBER_SLOTS_PER_TASK_MANAGER = 2;
     private static final int PARALLELISM = NUMBER_SLOTS_PER_TASK_MANAGER * NUMBER_TASK_MANAGERS;
 
     private static final Configuration configuration = getConfiguration();
 
     private static Configuration getConfiguration() {
-        final Configuration configuration = new Configuration();
+        final Configuration conf = new Configuration();
```

Comment:
I'm not sure whether the renaming is actually necessary here. I would propose reverting it for the sake of the git history.

Reply:
I renamed it because an IntelliJ inspection complained about a method field hiding a class field.